Evaluations_power_boxplots.ipynb

```diff
@@ -84,15 +84,40 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 7,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "About to retrieve messages using {'user': 'shankari@eecs.berkeley.edu', 'key_list': ['config/evaluation_spec'], 'start_time': 0, 'end_time': 1613685967}\n",
```

Comment:
Please don't check in outputs, at least for files that didn't have outputs to begin with.

Reply:
Don't change the XXX_master files that had outputs just yet. As you point out, we haven't figured out how to run the algorithms on in-memory data instead.

Reply:
Try and keep the PR as minimal as possible so it is easy to review and understand.